SmartDeviceLink/SDLAsynchronousRPCRequestOperation.m

```diff
@@ -90,7 +90,7 @@ - (void)sdl_sendRequests {
 
 - (void)sdl_sendRequest:(SDLRPCRequest *)request {
     __weak typeof(self) weakSelf = self;
-    [self.connectionManager sendConnectionRequest:request withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    [self.connectionManager sendConnectionRequest:(__kindof SDLRPCMessage *)request withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
```

Comment:
We should be able to just send a request because it is a subclass of `SDLRPCMessage` but Obj-C doesn't let us do that. It only works if all classes implement the same protocol.